risc0/zkvm/src/receipt.rs

```diff
@@ -362,20 +362,29 @@ impl Global {
     }
 }
 
-/// TODO
+/// A receipt attesting to the execution of a Session.
 #[derive(Deserialize, Serialize, Clone, Debug)]
 pub struct SessionReceipt {
-    /// TODO
+    /// The constituent [SegmentReceipt]s.
+    ///
+    /// Together these can be used by [SessionReceipt::verify] to
+    /// cryptographically prove that this full Session was faithfully executed.
     pub segments: Vec<SegmentReceipt>,
 
-    /// TODO
+    /// The public data written by the guest in this Session.
+    ///
+    /// This data is cryptographically attested to in [SessionReceipt::verify].
```

Comment:
"attested to in `::verify` " feels off to me.

Perhaps: "This data is cryptographically authenticated in [SessionReceipt::verify]."